docs/getting_started/notifications-Listing_Creation.md

```diff
@@ -43,6 +44,17 @@ For example, you can decide to trigger a notification when the following criteri
 4. Give your notification a `name`
 5. Save your changes
 
+
+!!! Note 
+    **Notifications concerning playbooks**
```

Comment:
@mathieubellon I was thinking maybe this section belongs to [this page](https://docs.sekoia.io/getting_started/notifications-Examples/)? Or deserves a section "Playbooks' notifications"